image/manifest.go

```diff
@@ -84,7 +85,7 @@ func manifestInstanceFromBlob(src types.ImageSource, manblob []byte, mt string)
 	// need to happen within the ImageSource.
 	case manifest.DockerV2Schema1MediaType, manifest.DockerV2Schema1SignedMediaType, "application/json":
 		return manifestSchema1FromManifest(manblob)
-	case manifest.DockerV2Schema2MediaType:
+	case manifest.DockerV2Schema2MediaType, imgspecv1.MediaTypeImageManifest:
```

Comment:
Fair enough. Consider at least adding a comment here, saying that `docker_schema2.go` is good enough for reading images, but this will need to be modified for write support due to differing MIME types.